pkgs/development/python-modules/ptyprocess/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "ptyprocess";
+  version = "0.5.2";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256= "e64193f0047ad603b71f202332ab5527c5e52aa7c8b609704fc28c0dc20c4365";
```

Comment:
Missing blank.
And why don't you use short hash? `0ra31k10v3629xq0kdn8lwmfbi97anmk48r03yvh7mks0kq96hg6`

Reply:
Simple copy-paste from PyPI ;-)

Reply:
I must be blind. Where did you find sha256? I only see md5

Reply:
https://pypi.org/project/ptyprocess/#files